playground/lgsvl/placeCarOnJunction/main.py

```diff
@@ -17,7 +17,7 @@ def _main() -> None:
     # Setup simulation
     sim = Simulator()
     load_scene(sim, SupportedMap.BorregasAve)
-    initial_state = generate_initial_state(get_entry_point(road_points, random_connection))
+    initial_state = generate_initial_state(get_entry_final_point(road_points, random_connection))
```

Comment:
I encountered an error `distance` argument is missing. As far as I can see, the value might be from 0 to less than 1.

Reply:
placeCarOnJunction was just a pre step. The current project is executeRandomTests. placeCarOnJunction may thus not work since it's not a product.